### ISSUE TITLE
Feature: Handwritten note in printed text

### ISSUE THREAD

**Issue by StefanDumont** [deutschestextarchiv/dtabf#17]
In the project "Travelling Humboldt – Science on the Move" we have some _printed_ texts which were later annotated by Alexander von Humboldt or other persons. Therefore we need the possibility to encode such handwritten annotations. Suggestion: 

`<note type="handwritten"></note>`

**Comment by cthomasdta** [comment]
Ist das so Beschluss-fähig, für DTABf und ehd? Dann müssten wir es auch in beiden dokumentieren -- ehd hätte in dem Fall gewissermaßen höhere Dringlichkeit, weil es dort ja so schon benutzt wird. Vgl. https://edition-humboldt.de/H0002731 Bl. 9r und Bl. 10r.

**Comment by FrederikeNeuber** [comment]
Ist mit Angabe von `@hand` nicht ohnehin klar, dass es sich um etwas handschriftlich verfasstes handelt? Bzw. wenn mit `@rendition="#mPrint"` die Tatsache, dass es sich um etwas gedrucktes handelt, ausgezeichnet wird, ist es dann nicht konsistenter im anderen Fall eben `"#mHandwritten"` (was auch immer das "m" heißt) zu verwenden? 
Ich finde mit `note/@type='handwritten'` wird auch die Typisierung der note-Elemente inkonsistent. Es gibt ja auch `note/@type='editorial'`. Die Typisierung findet dann auf ganz anderen Ebenen statt, einmal geht es um die Schreibtechnik, das andere mal um eine Kategorisierung der Verfasserschaft.

**Comment by burki** [comment]
ich finde wie @FrederikeNeuber  auch `@type="handwritten"` nicht kongruent zu `@type="editorial"`, da es mehr auf das "wie" (handwritten) als auf das "in welchem Kontext wurde die note eingeführt" zielt.
Ich bin aber dafür, einen neuen type einzuführen, der markiert, dass dies so nicht im Original steht, aber auch nicht im Editorial-Prozess vergeben wurde.

**Comment by cthomasdta** [comment]
Ich stimme aus pragmatischen Gründen zu (da wir es in ehd schon verwenden), es sollte ein spezifischer `@type`-Wert sein. Der kann gern anders als `"handwritten"` heißen. Vorschläge?

**Comment by cthomasdta** [comment]
Insbesondere hier würde ich auch @StefanDumont um Einschätzung bitten.
